examples/bpk-component-accordion/stories.tsx

```diff
@@ -78,6 +78,7 @@ export const VisualTest = SingleItemExample;
 export const VisualTestOnDark = WithDarkBackgroundExample;
 
 export const VisualTestWithZoom = VisualTest.bind({});
+// @ts-expect-error failed typings
 VisualTestWithZoom.args = {
   zoomEnabled: true,
 };
```

Comment:
I believe this can be fixed by doing

```suggestion
export const VisualTestWithZoom = {
  render: VisualTest,
  args: {
    zoomEnabled: true,
  }
};
```

This is what we have done with other TS converted stories :)